watch_service_test.go

```diff
@@ -256,7 +256,9 @@ func TestWatcher_started_AwaitInitialValue(t *testing.T) {
 		}
 
 		latest2 := fw.Latest()
-		if !reflect.DeepEqual(latest2, latest) {
+		if !reflect.DeepEqual(latest2.Entry, latest.Entry) ||
+			latest2.HttpStatusCode != latest.HttpStatusCode ||
+			latest2.Revision != latest.Revision {
```

Comment:
latest is WatchResult type. Which contains error -> linter warning of reflect.DeepEqual on Err type